src/pybind/mgr/zabbix/module.py

```diff
@@ -88,12 +88,8 @@ def __init__(self, *args, **kwargs):
     def init_module_config(self):
         for key, default in self.config_keys.items():
             value = self.get_localized_config(key, default)
```

Comment:
The `get_localized_config` should be returning the default if the value has not been set.  If it's not doing that, then there's some other bug happening here.

So for e.g. `interval` not being set by the user, the `default` argument here would be 60 (see self.config_keys definition), and `value` would be set to 60 as well.

Reply:
Indeed, the func get_localized_config return the default value. But when it met the key zabbix_host,which we do not set yet, will raise Exception while not continue for loop .Then later default key will be set .